### PR TITLE
Use {{ ansible_managed | comment }} to fix multi-line ansible_managed

### PR DESCRIPTION
**Pull request by spetrosi**
BZ#2006230, BZ#2006231, BZ#2006233

### CODE REVIEW

File: templates/sshd_config.j2 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# {{ ansible_managed }}
+{{ ansible_managed | comment }}
```
Comment (Jakuje) [inline templates/sshd_config.j2:1]: please, update also the https://github.com/willshersystems/ansible-sshd/blob/master/meta/01_ansible_head.j2 which generates this file

Comment (spetrosi) [reply templates/sshd_config.j2:1]: thanks, done

**Review by Jakuje — APPROVED** [review]

**Comment by spetrosi** [comment]
@mattwillsher Can we merge this PR?

**Comment by mattwillsher** [comment]
Done. Will release shortly. Thank you!

**Pull request merged**